.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/GitVersionTfsTask/BuildScripts"]
```

Comment:
We've decided to go forward with the npm package. So I've updated this PR accordingly. I'll investigate the vsts beta feature.